src/space.cpp

```diff
@@ -1073,6 +1073,10 @@ static iarf_e do_space(chunk_t *first, chunk_t *second, int &min_sp)
       {
          return(IARF_FORCE);
       }
+      if (chunk_is_token(first, CT_ASM_COLON) && options::sp_before_square_ignore_asm_block())
```

Comment:
there's nothing saying here that we are in a macro

Reply:
CT_ASM_COLON in first means we are in a C ASM macro. Otherwise another token would have been found. Should I add a comment?